db/migrate/20171011225553_create_host_genomes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateHostGenomes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :host_genomes do |t|
+      t.string :name, null:false, unique: true
+      t.text :s3_star_index_path
+      t.text :s3_bowtie2_index_path
+      t.bigint :default_background_id
```

Comment:
is this a circular dependency?

  -- in order to create background, you need several runs

  -- for each run, you need a host genome

  -- for a host genome, you need a background?

Reply:
well that's why it's called default_background_id. 

For each host genome, there would be a background that consists of runs that are run based on the host genome, which has the indexes that areapecified in the beginning. 

Once those are done, a default could be created but it's not there in the beginning.